propagators/opentelemetry-propagator-instana/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    "env": {
+        "mocha": true,
+        "commonjs": true,
```

Comment:
Wondering why we use "commonjs": true, because we use ts esm.

Reply:
To be honest, I simply used another propagator (ot-trace) as a template, in particular for the boilerplate/config files. But you're right, commonjs doesn't make sense, I removed it.